### DISCUSSION TITLE
Is it possible to use Bacalhau in service in production?

### DISCUSSION THREAD

**Discussion by israelmarmar** [bacalhau-project/bacalhau#2005] [Q&A]
Hello, I would like to use Bacalhau in a SaaS, but I wanted to know if the project has limitation of requests by us or if it is possible to pay for the extra processing in Filecoins

**Comment by aronchick** [comment]
Hiya! I'd love to chat more about your needs - can you reach out in slack? I'm "David Aronchick" in here (https://bit.ly/bacalhau-slack)

Or you can just email me at david.aronchick@protocol.ai

**Reply by it09** [reply]
Hi @israelmarmar have you had a chance to reach out to David or you need help? We want to learn about what you are doing and how we can support you
